spyder/widgets/variableexplorer/dataframeeditor.py

```diff
@@ -15,13 +15,15 @@
 
 # Third party imports
 from pandas import DataFrame, Series
+from qtpy import PYQT5

```

Comment:
Move this import to the line below, like this

    from qtpy import API, PYQT5